build/BundledDotnetTools.props

```diff
@@ -1,8 +1,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <BundledDotnetTools Include="dotnet-dev-certs" Version="$(AspNetCoreVersion)" />
-    <BundledDotnetTools Include="dotnet-sql-cache" Version="$(AspNetCoreVersion)" />
-    <BundledDotnetTools Include="dotnet-user-secrets" Version="$(AspNetCoreVersion)" />
-    <BundledDotnetTools Include="dotnet-watch" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-sql-cache" Version="$(AspNetCoreVersion)" ObsoletesCliTool="Microsoft.Extensions.Caching.SqlConfig.Tools" />
```

Comment:
This metadata name reads weird. But then again, English is not my primary language.

Reply:
Reads fine to me 😜